mayastor-test/e2e/basic_volume_io/basic_volume_io_test.go

```diff
@@ -0,0 +1,73 @@
+// JIRA: CAS-505
+// JIRA: CAS-506
```

Comment:
Drop these comments.

Reply:
IIRC these are required so that we know which JIRAs the test is for.

Reply:
OK, I see 2 other tests have them too (the first 2 tests I initially looked at didn't).